package.json

```diff
@@ -29,28 +30,26 @@
     "url": "https://github.com/ipfs/is-ipfs.git"
   },
   "scripts": {
+    "prepare": "aegir build --no-bundle",
```

Comment:
What does `--no-bundle` do?

Reply:
It skips the browser bundle step as you don't need it during development, i.e. it only generates types from the jsdoc comments.